src/main/java/com/animeviewing/application/repositories/AnimationRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.animeviewing.application.repositories;
+
+import java.util.List;
+
+import org.springframework.boot.autoconfigure.data.web.SpringDataWebProperties.Pageable;
+
+import com.animeviewing.application.entities.Animation;
+
+/**
+ * アニメーションリポジトリークラス
+ */
+public interface AnimationRepository {
+	/**
+	 * アニメ一覧取得
+	 */
+	public List<Animation> findAll(Pageable pageable);
```

Comment:
こちらですが、モデルや、サービスなんかはシステムのコアとなる部分なので、できるだけライブラリに頼った実装にしたくないです。なので、以下のように修正したいです。

```suggestion
	public List<Animation> findAll(Integer limit, Integer offset);
```

他も一緒ですかね。

Reply:
確認したのでリゾルブ